.evergreen/csfle/kms_failpoint_server.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+import http.server
+import json
+import urllib
+import ssl
+import base64
+
+"""
+Example of setting a network failpoint:
+
+$ curl -X POST localhost:3000
+{"message": "ok"}
+
+$ curl -X GET localhost:3000/set_failpoint/network/1
+{"message": "failpoint set for type: 'network'"}
+
+$ curl -X POST localhost:3000
+curl: (52) Empty reply from server
+
+$ curl -X POST localhost:3000
+{"message": "ok"}
+
+"""
+
+# A new instance of Hander is created for every request, so these have to be global variables
+failpoint_type = None
```

Comment:
Suggest removing `failpoint_type` as a global. It is only used to determine which failpoint to set in `do_GET`.